### PR TITLE
[FrameworkBundle][TwigBridge] do not render empty form action attributes

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #13852, #15995 |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

**Comment by fabpot** [comment]
:+1:

**Comment by stof** [comment]
this should be covered by a test to avoid regressions

**Comment by xabbuh** [comment]
tests added

**Comment by fabpot** [comment]
Thank you @xabbuh.

File: src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig [hunk 1]
```diff
@@ -274,7 +274,7 @@
     {%- else -%}
         {% set form_method = "POST" %}
     {%- endif -%}
-    <form name="{{ name }}" method="{{ form_method|lower }}" action="{{ action }}"{% for attrname, attrvalue in attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}{% if multipart %} enctype="multipart/form-data"{% endif %}>
+    <form name="{{ name }}" method="{{ form_method|lower }}"{% if action %} action="{{ action }}"{% endif %}{% for attrname, attrvalue in attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}{% if multipart %} enctype="multipart/form-data"{% endif %}>
```
Comment (Tobion) [inline src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig]: this wil fail for action="0" again, which would be a valid relative path

**Pull request merged**